src/Umbraco.Web.UI.Client/src/packages/documents/document-types/entity-actions/import/modal/document-type-import-modal.element.ts

```diff
@@ -6,7 +6,7 @@ import type {
 import { css, html, customElement, query, state, when } from '@umbraco-cms/backoffice/external/lit';
 import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
 import { UmbModalBaseElement } from '@umbraco-cms/backoffice/modal';
-import type { UmbDropzoneElement } from '@umbraco-cms/backoffice/media';
+import type { UmbDropzoneElement } from '@umbraco-cms/backoffice/dropzone';
```

Comment:
In theory, moving an element to another import map would be breaking, but I would assume no one uses this component. In our current process, we need a bit of flexibility, so all good. Just be aware as we get closer to v.17

Reply:
Oh, I see it is already re-exported. Then everything is so fine. I'll resolve this comment.